web/cobrands/peterborough/js.js

```diff
@@ -0,0 +1,42 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var defaults = {
+    http_options: {
+        url: "https://tilma.staging.mysociety.org/mapserver/peterborough",
+        params: {
+            SERVICE: "WFS",
+            VERSION: "1.1.0",
+            REQUEST: "GetFeature",
+            SRSNAME: "urn:ogc:def:crs:EPSG::3857"
+        }
+    },
+    format_class: OpenLayers.Format.GML.v3.MultiCurveFix, // Not sure needed any more
```

Comment:
We have MapServer 7 on the server now, so I assume this *isn't* needed? Would be good to find an example geometry to confirm this for sure.

Reply:
Yep, looks as though our mapserver is doing the right thing now. See, e.g. geometry of the second `featureMember` here: https://tilma.mysociety.org/mapserver/highways?SERVICE=WFS&VERSION=1.1.0&REQUEST=GetFeature&SRSNAME=urn%3Aogc%3Adef%3Acrs%3AEPSG%3A%3A3857&TYPENAME=Highways&bbox=-35748.263942009,6687326.5660957,-34175.212234679,6688378.3808725